.travis.yml

```diff
@@ -17,5 +18,5 @@ script:
   - '! gofmt -s -d -l . 2>&1 | read'
   - '! goimports -l . | read'
   - '! golint ./... | grep -vE "(_string|\.pb)\.go:"'
-  - '! go tool vet -all . 2>&1 | grep -vE "constant [0-9]+ not a string in call to Errorf"'
+  - '! go tool vet -all -composites=false . 2>&1 | grep -vE "constant [0-9]+ not a string in call to Errorf"'
```

Comment:
a "gentler" way to achieve this is to append ` | grep -vF 'pb.go:'` to the end of this line.